.github/workflows/master_build_merge.yml

```diff
@@ -206,6 +211,48 @@ jobs:
             docker push ${DOCKERHUB_ORG}/$IMAGE:${LATEST_TAG}
             set +e
           }
+          elasticBuild() {
+            set -e
+            IMAGE="$1"
+            git config --global url."https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com".insteadOf "https://github.com"
+            go version
+            go install golang.org/x/tools/cmd/goimports@latest
+            which go
+            git clone https://github.com/magefile/mage
+            cd mage
+            go run bootstrap.go
+            cd ..
+            echo "----------------------------------------------"
+            echo "----------- Building Service -----------------"
+            export GOPATH=/home/runner/go/bin
+            export PATH=/go/bin:/usr/local/go/bin:/opt/maven/bin:/usr/lib/jvm/java-16-openjdk-amd64/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/runner/go/bin
```

Comment:
This path is incorrect like for e.g.,
1. It has JVM 16 path, which is not needed.
2. It should use env variable rather than hardcoding `/home/runner/go/bin`
3. Where is `/go/bin` coming from? Are we installing it? Same with /usr/local/go/bin, maven, etce
4.